### PR TITLE
S2U-42 Card Game: Force 1:1 aspect ratio for image

### PR DESCRIPTION
**Pull request by SirSaibot**
Jira: https://sakaiproject.atlassian.net/browse/S2U-42

### CODE REVIEW

File: library/src/morpheus-master/sass/modules/_b5-polyfill.scss [hunk 1]
```diff
@@ -12,6 +12,29 @@ TODO Master merge: Remove this file
 }
 
 .b5 {
+  &.ratio{
```
Comment (stetsche) [inline library/src/morpheus-master/sass/modules/_b5-polyfill.scss:15]: could you test this using the scss nesting please?
```
.ratio {
    position: relative;
    width: 100%
    
    & > * {
        position: absolute;
        left: 0;
        top: 0;
        width: 100%;
        height: 100%;
    }

    &::before {
        display: block;
        padding-top: var(--bs-aspect-ratio);
        content: "";
    }
}
```

File: library/src/morpheus-master/sass/modules/tool/roster/_roster.scss [hunk 2]
```diff
@@ -240,6 +240,11 @@
             background-color: var(--sakai-primary-color-1);
         }
 
+        //keep image from streching when not having 1:1 aspect ratio
```
Comment (stetsche) [inline library/src/morpheus-master/sass/modules/tool/roster/_roster.scss:243]:
```suggestion
        // Keep image from stretching when not having 1:1 aspect ratio
```

File: library/src/morpheus-master/sass/modules/tool/roster/_roster.scss [hunk 3]
```diff
@@ -240,6 +240,11 @@
...
+        .img-ratio-cover{
```
Comment (stetsche) [inline library/src/morpheus-master/sass/modules/tool/roster/_roster.scss:244]: spacing
```suggestion
        .img-ratio-cover {
```

**Review by stetsche — CHANGES_REQUESTED** [review]
Looks pretty good, I requested some minor changes. Good job so far!

**Review by stetsche — APPROVED** [review]
Looks good to me now!

**Comment by mpellicer** [comment]
Thanks @SirSaibot you're a monster!

**Pull request merged**